Modules/Core/Support/MailerHelper.php

```
@@ -93,4 +93,139 @@ public static function check_mail_errors(array $errors = [], $redirect = ''): vo
+
+        if ($bcc && ! self::validate_email_address($bcc)) {
+            $errors[] = 'bcc_email';
+        }
+
+        self::check_mail_errors($errors, 'mailer/quote/' . $quote_id);
+
+        $message = (empty($message) ? ' ' : $message);
+
+        // Use the namespaced phpmail_send function
+        return \Modules\Sessions\Controllers\phpmail_send($from, $to, $subject, $message, $quote, $cc, $bcc, $attachments);
+    }
```

PHPMD (2.15.0)

The method email_quote is not named in camelCase. (undefined)

(CamelCaseMethodName)